--- origsrc/pan-0.133/configure.in	2008-07-29 21:00:01.000000000 -0500
+++ src/pan-0.133/configure.in	2008-08-14 02:59:57.098240800 -0500
@@ -31,6 +31,7 @@
 AC_HEADER_STDC
 AC_HEADER_TIME
 AC_CHECK_FUNCS([localtime_r])
+AC_SEARCH_LIBS(getaddrinfo, getaddrinfo)
 AC_CXX_HEADER_TR1_UNORDERED_SET
 AC_CXX_HAVE_EXT_HASH_SET
 AC_PROG_AWK
--- origsrc/pan-0.133/pan/tasks/socket-impl-gio.cc	2008-07-05 01:38:58.000000000 -0500
+++ src/pan-0.133/pan/tasks/socket-impl-gio.cc	2008-08-14 02:58:42.723240800 -0500
@@ -81,6 +81,10 @@
   #include <netdb.h>
   #include <arpa/inet.h>
   #define closesocket(fd) close(fd)
+  #ifdef __CYGWIN__
+    #include <dummyin6.h>
+    #include <getaddrinfo.h>
+  #endif
 #endif
 
 #include <pan/general/debug.h>
