--- origsrc/pan-0.132/configure.in	2007-08-01 12:36:15.000000000 -0500
+++ src/pan-0.132/configure.in	2007-08-05 00:42:37.000000000 -0500
@@ -40,6 +40,10 @@
 dnl for libuu
 AC_CHECK_HEADERS([errno.h fcntl.h])
 
+AC_SEARCH_LIBS(gai_strerror, getaddrinfo,
+   [AC_DEFINE(NEED_GETADDRINFO_H, 1, [Define if building against external getaddrinfo])],
+   [AC_MSG_ERROR([gai_strerror required for building pan])])
+
 AC_PROG_INTLTOOL([0.23],[no-xml])
 GETTEXT_PACKAGE=pan
 AC_SUBST(GETTEXT_PACKAGE)
@@ -91,7 +95,7 @@
 	AC_DEFINE(G_OS_DARWIN,[1],[Darwin])
 	;;
 
-	*cygwin|*mingw32*)
+	*mingw32*)
 	CXXFLAGS="$CXXFLAGS -mms-bitfields -mwin32 -mwindows"
 	CPPFLAGS="$CPPFLAGS -DWIN32_LEAN_AND_MEAN"
 	LIBS="$LIBS -lshell32 -lws2_32"
--- origsrc/pan-0.132/pan/tasks/socket-impl-gio.cc	2007-08-01 12:00:00.000000000 -0500
+++ src/pan-0.132/pan/tasks/socket-impl-gio.cc	2007-08-05 00:42:37.015625000 -0500
@@ -33,6 +33,11 @@
   #include <unistd.h>
   #include <glib.h>
   #include <glib/gi18n.h>
+
+  #ifdef NEED_GETADDRINFO_H
+  #include <dummyin6.h>
+  #include <getaddrinfo.h>
+  #endif
 }
 
 #include <pan/general/file-util.h>
--- origsrc/pan-0.132/pan/usenet-utils/Makefile.am	2007-08-01 11:59:59.000000000 -0500
+++ src/pan-0.132/pan/usenet-utils/Makefile.am	2007-08-05 00:42:37.015625000 -0500
@@ -27,7 +27,6 @@
  gnksa-test \
  message-check-test \
  numbers-test \
- scorefile-test \
  text-massager-test \
  url-find-test
 
@@ -39,8 +38,6 @@
 message_check_test_LDADD = $(TEST_LDADD)
 numbers_test_SOURCES = numbers-test.cc
 numbers_test_LDADD = $(TEST_LDADD)
-scorefile_test_SOURCES = scorefile-test.cc
-scorefile_test_LDADD = $(TEST_LDADD)
 text_massager_test_SOURCES = text-massager-test.cc
 text_massager_test_LDADD = $(TEST_LDADD)
 url_find_test_SOURCES = url-find-test.cc
